### ISSUE TITLE
The last match still shows winner's quote 

### ISSUE THREAD

**Issue by shadowpage** [ikemen-engine/Ikemen-GO#1605]
### Describe the bug

1. If you won the last match, Ikemen still shows winner's quote
2. Game Over show twice at the end of the game.

### To Reproduce

Play the arcade mode till the end.

### Expected behavior

1. If you won the last match, Mugen won't show winner's quote
2. Game Over should show only once.

### Engine Version (or source code date)

v0.99.0

### Operating system

None

**Comment by K4thos** [comment]
1. It's intentional
2. I can't reproduce the issue. It seems you might have set the game over screen as the credits storyboard by mistake. However, there was a separate issue: setting gameover.enabled = 0 under [Continue Screen] disabled the game over screen for the last match, not just the continue screen. This has been fixed in #1646
